qa/evil-tests/src/test/java/org/elasticsearch/plugins/ListPluginsCommandTests.java

```diff
@@ -66,18 +76,150 @@ public void testNoPlugins() throws Exception {
+                "java.version", System.getProperty("java.specification.version"),
+                "classname", "org.fake");
         MockTerminal terminal = listPlugins(env);
         String output = terminal.getOutput();
         assertTrue(output, output.contains("fake1"));
         assertTrue(output, output.contains("fake2"));
     }
+    
+    public void testPluginWithVerbose() throws Exception {
+        Environment env = createEnv();
+        PluginTestUtil.writeProperties(env.pluginsFile().resolve("fake1"),
+                "description", "fake desc",
+                "name", "fake_plugin",
+                "version", "1.0",
+                "elasticsearch.version", Version.CURRENT.toString(),
+                "java.version", System.getProperty("java.specification.version"),
+                "classname", "org.fake");
+        String[] params = { "-v" };
+        MockTerminal terminal = listPlugins(env, params);
+        String output = terminal.getOutput();
+        assertTrue(output, output.contains("Plugin information"));
+        assertTrue(output, output.contains("Classname: org.fake"));
+        assertTrue(output, output.contains("fake_plugin"));
+        assertTrue(output, output.contains("1.0"));
+        assertTrue(output, output.contains("fake desc"));
+    }
+    
+    public void testPluginWithVerboseMultiplePlugins() throws Exception {
+        Environment env = createEnv();
+        
+        PluginTestUtil.writeProperties(env.pluginsFile().resolve("fake1"),
+                "description", "fake desc",
+                "name", "fake_plugin",
+                "version", "1.0",
+                "elasticsearch.version", Version.CURRENT.toString(),
+                "java.version", System.getProperty("java.specification.version"),
+                "classname", "org.fake");
+
+        PluginTestUtil.writeProperties(env.pluginsFile().resolve("fake2"),
+                "description", "fake desc",
+                "name", "fake2_plugin",
+                "version", "1.0",
+                "elasticsearch.version", Version.CURRENT.toString(),
+                "java.version", System.getProperty("java.specification.version"),
+                "classname", "org.fake2");
+        
+        PluginTestUtil.writeProperties(env.pluginsFile().resolve("fake3"),
+                "description", "fake desc",
+                "name", "fake3_plugin",
+                "version", "1.0",
+                "elasticsearch.version", Version.CURRENT.toString(),
+                "java.version", System.getProperty("java.specification.version"),
+                "classname", "org.fake3");
+
+        String[] params = { "-v" };
+        MockTerminal terminal = listPlugins(env, params);
+        String output = terminal.getOutput();
+        
+        //Should be 3 plugins, so if i split by plugin informations i should get 4 strings
+        assertTrue(output, output.split("Plugin information").length == 4);
```

Comment:
I will change some of the assertions to use assertEquals. The message is actually much better than assert True.